thespian/system/transport/asyncTransportBase.py

```diff
@@ -35,9 +35,9 @@
 # transmits are immediately failed instead of being queued.
 
 MAX_PENDING_TRANSMITS = 20
-MAX_QUEUED_TRANSMITS = 950
-QUEUE_TRANSMIT_UNBLOCK_THRESHOLD = 780
-DROP_TRANSMITS_LEVEL = MAX_QUEUED_TRANSMITS + 100
+MAX_QUEUED_TRANSMITS = int(os.environ.get("THESPIAN_MAX_QUEUED_TRANSMITS", 950))
+QUEUE_TRANSMIT_UNBLOCK_THRESHOLD = int(os.environ.get("THESPIAN_QUEUED_TRANSMIT_UNBLOCK_THRESHOLD", 780))
+DROP_TRANSMITS_LEVEL = MAX_QUEUED_TRANSMITS + int(os.environ.get("THESPIAN_DROP_TRANSMITS_LEVEL", 100))
```

Comment:
I would expect `THESPIAN_DROP_TRANSMITS_LEVEL` to determine the final value of `DROP_TRANSMITS_LEVEL`, not part of it.

Reply:
Can we add `MAX_PENDING_TRANSMITS` env var for completeness?

Reply:
> Can we add `MAX_PENDING_TRANSMITS` env var for completeness?

Yes.  Didn't know about that one

Reply:
> > I would expect `THESPIAN_DROP_TRANSMITS_LEVEL` to determine the final value of `DROP_TRANSMITS_LEVEL`, not part of it.
> 
> Yeah, that seems badly named since it's a calculated value - but that's how it was getting set in the original code so I didn't want to mess with it too much. It should either be the value of the env var, or be named something different. I'm fine with whichever you think makes the most sense

The intent of the original code was to ensure that the `DROP_TRANSMITS_LEVEL` always exceeded `MAX_QUEUED_TRANSMITS` to most effectively use the various fallback capabilities.  In general, there's a danger here that overrides for any of these values could create significantly sub-optimal behavior and ideally there would be a lot more validation and behavioral checks.  That said, all parties on this PR and anyone likely to provide these overrides are expert users to whom control should be ceded rather than second-guessed.  The original values for all of these parameters were "tuned" to meet some original configurations anyhow, so none of these should be considered immutable or omniscient.

I agree that when setting a value, it should probably be set to directly the value specified rather than a modified version of that value.